src/software/geom/rectangle.h

```diff
@@ -24,26 +24,26 @@ class Rectangle final
      *
      * @param neg_x_neg_y_corner The <-x,-y> corner of the rectangle
      *
-     * @param width The width of the rectangle
+     * @param xLength The xLength of the rectangle
      *
-     * @param height The height of the rectangle
+     * @param yLength The yLength of the rectangle
      */
-    explicit constexpr Rectangle(const Point &neg_x_neg_y_corner, double width,
-                                 double height);
+    explicit constexpr Rectangle(const Point &sw_corner, double x_length, double y_length);
+
 
     /**
-     * Returns the width of the rectangle
+     * Returns the horizontal length of the rectangle
      *
-     * @return The width of the rectangle
+     * @return The horizontal length of the rectangle
      */
-    constexpr double width() const;
+    constexpr double xLength() const;
 
     /**
-     * Returns the height of the rectangle
+     * Returns the vertical length of the rectangle
```

Comment:
We should probably replace "vertical length" with "length along the y-axis" or something similar.